### ISSUE TITLE
AppVeyor fails when doing Choco Pack: 

### ISSUE THREAD

**Issue by KirillOsenkov** [chocolatey/choco#1402]
### What You Are Seeing?

This AppVeyor build fails when doing Choco Pack:
https://ci.appveyor.com/project/KirillOsenkov/msbuildstructuredlog/build/1.1.178
`File specified is either not found or not a .nuspec file.`

The failure originates here:
https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/services/NugetService.cs#L252-L253

### What is Expected?
It passes locally and I think that both the file should exist and the extension is .nuspec.

### How Did You Get This To Happen? (Steps to Reproduce)
Just build in AppVeyor (passes locally).

### Output Log
```
Chocolatey v0.10.7
  Chocolatey is running on Windows v 10.0.14393.0
  Attempting to delete file "C:/ProgramData/chocolatey/choco.exe.old".
  Attempting to delete file "C:\ProgramData\chocolatey\choco.exe.old".
  Command line: C:\ProgramData\chocolatey\choco.exe  Pack C:\projects\msbuildstructuredlog\src\StructuredLogViewer\msbuild-structured-log-viewer.nuspec -OutputDirectory C:\projects\msbuildstructuredlog\bin\Release\ -prop currentVersion=1.1.178 outdir=C:\projects\msbuildstructuredlog\bin\Release\ --debug --verbose
  Received arguments: Pack C:\projects\msbuildstructuredlog\src\StructuredLogViewer\msbuild-structured-log-viewer.nuspec -OutputDirectory C:\projects\msbuildstructuredlog\bin\Release\ -prop currentVersion=1.1.178 outdir=C:\projects\msbuildstructuredlog\bin\Release\ --debug --verbose
  RemovePendingPackagesTask is now ready and waiting for PreRunMessage.
  Sending message 'PreRunMessage' out if there are subscribers...
  [Pending] Removing all pending packages that should not be considered installed...
  The source 'https://chocolatey.org/api/v2/' evaluated to a 'normal' source type
  
  NOTE: Hiding sensitive configuration data! Please double and triple 
   check to be sure no sensitive data is shown, especially if copying 
   output to a gist for review.
  Configuration: CommandName='Pack'|
  CacheLocation='C:\Users\appveyor\AppData\Local\Temp\1\chocolatey'|
  ContainsLegacyPackageInstalls='True'|
  CommandExecutionTimeoutSeconds='2700'|WebRequestTimeoutSeconds='30'|
  Sources='https://chocolatey.org/api/v2/'|SourceType='normal'|
  Debug='True'|Verbose='True'|Trace='False'|Force='False'|Noop='False'|
  HelpRequested='False'|UnsuccessfulParsing='False'|RegularOutput='True'|
  QuietOutput='False'|PromptForConfirmation='False'|
  AcceptLicense='False'|
  AllowUnofficialBuild='False'|
  Input='C:\projects\msbuildstructuredlog\src\StructuredLogViewer\msbuild-structured-log-viewer.nuspec -prop currentVersion=1.1.178 outdir=C:\projects\msbuildstructuredlog\bin\Release\'|
  
  AllVersions='False'|SkipPackageInstallProvider='False'|
  OutputDirectory='C:\projects\msbuildstructuredlog\bin\Release\'|
  Prerelease='False'|ForceX86='False'|OverrideArguments='False'|
  NotSilent='False'|ApplyPackageParametersToDependencies='False'|
  ApplyInstallArgumentsToDependencies='False'|IgnoreDependencies='False'|
  AllowMultipleVersions='False'|AllowDowngrade='False'|
  ForceDependencies='False'|Information.PlatformType='Windows'|
  Information.PlatformVersion='10.0.14393.0'|
  Information.PlatformName='Windows Server 2016'|
  Information.ChocolateyVersion='0.10.7.0'|
  Information.ChocolateyProductVersion='0.10.7'|
  Information.FullName='choco, Version=0.10.7.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|
  
  Information.Is64BitOperatingSystem='True'|
  Information.Is64BitProcess='True'|Information.IsInteractive='True'|
  Information.UserName='appveyor'|
  Information.UserDomainName='MASTER-WIN2016'|
  Information.IsUserAdministrator='True'|
  Information.IsUserSystemAccount='False'|
  Information.IsUserRemoteDesktop='False'|
  Information.IsUserRemote='False'|Information.IsProcessElevated='True'|
  Information.IsLicensedVersion='False'|Information.LicenseType='Foss'|
  Features.AutoUninstaller='True'|Features.ChecksumFiles='True'|
  Features.AllowEmptyChecksums='False'|
  Features.AllowEmptyChecksumsSecure='True'|
  Features.FailOnAutoUninstaller='False'|
  Features.FailOnStandardError='False'|Features.UsePowerShellHost='True'|
  Features.LogEnvironmentValues='False'|Features.VirusCheck='False'|
  Features.FailOnInvalidOrMissingLicense='False'|
  Features.IgnoreInvalidOptionsSwitches='True'|
  Features.UsePackageExitCodes='True'|
  Features.UseFipsCompliantChecksums='False'|
  Features.ShowNonElevatedWarnings='True'|
  Features.ShowDownloadProgress='True'|
  Features.StopOnFirstPackageFailure='False'|
  Features.UseRememberedArgumentsForUpgrades='False'|
  Features.ScriptsCheckLastExitCode='False'|
  ListCommand.LocalOnly='False'|
  ListCommand.IdOnly='False'|ListCommand.IncludeRegistryPrograms='False'|
  ListCommand.PageSize='25'|ListCommand.Exact='False'|
  ListCommand.ByIdOnly='False'|ListCommand.ByTagOnly='False'|
  ListCommand.IdStartsWith='False'|ListCommand.OrderByPopularity='False'|
  ListCommand.ApprovedOnly='False'|
  ListCommand.DownloadCacheAvailable='False'|
  ListCommand.NotBroken='False'|
  ListCommand.IncludeVersionOverrides='False'|
  UpgradeCommand.FailOnUnfound='False'|
  UpgradeCommand.FailOnNotInstalled='False'|
  UpgradeCommand.NotifyOnlyAvailableUpgrades='False'|
  UpgradeCommand.ExcludePrerelease='False'|
  NewCommand.AutomaticPackage='False'|
  NewCommand.UseOriginalTemplate='False'|SourceCommand.Command='unknown'|
  SourceCommand.Priority='0'|SourceCommand.BypassProxy='False'|
  SourceCommand.AllowSelfService='False'|
  FeatureCommand.Command='unknown'|
  ConfigCommand.Command='unknown'|PinCommand.Command='unknown'|
  OutdatedCommand.IgnorePinned='False'|Proxy.BypassOnLocal='True'|
  _ Chocolatey:ChocolateyPackCommand - Normal Run Mode _
EXEC : Chocolatey had an error occur:  [C:\projects\msbuildstructuredlog\src\StructuredLogViewer\StructuredLogViewer.csproj]
  System.ArgumentException: File specified is either not found or not a .nuspec file. 'C:\projects\msbuildstructuredlog\src\StructuredLogViewer\msbuild-structured-log-viewer.nuspec -prop currentVersion=1.1.178 outdir=C:\projects\msbuildstructuredlog\bin\Release\'
     at chocolatey.infrastructure.app.services.NugetService.<>c__DisplayClass23.<validate_and_return_package_file>b__22(String name, String value) in c:\codelocal\chocolatey2\src\chocolatey\infrastructure.app\services\NugetService.cs:line 253
     at chocolatey.infrastructure.guards.Ensure`1.meets(Func`2 ensureFunction, Action`2 exceptionAction) in c:\codelocal\chocolatey2\src\chocolatey\infrastructure\guards\Ensure.cs:line 83
     at chocolatey.infrastructure.app.services.NugetService.validate_and_return_package_file(ChocolateyConfiguration config, String extension) in c:\codelocal\chocolatey2\src\chocolatey\infrastructure.app\services\NugetService.cs:line 255
     at chocolatey.infrastructure.app.services.NugetService.pack_run(ChocolateyConfiguration config) in c:\codelocal\chocolatey2\src\chocolatey\infrastructure.app\services\NugetService.cs:line 260
     at chocolatey.infrastructure.app.services.ChocolateyPackageService.pack_run(ChocolateyConfiguration config) in c:\codelocal\chocolatey2\src\chocolatey\infrastructure.app\services\ChocolateyPackageService.cs:line 258
     at chocolatey.infrastructure.app.runners.GenericRunner.run(ChocolateyConfiguration config, Container container, Boolean isConsole, Action`1 parseArgs) in c:\codelocal\chocolatey2\src\chocolatey\infrastructure.app\runners\GenericRunner.cs:line 164
     at chocolatey.infrastructure.app.runners.ConsoleApplication.run(String[] args, ChocolateyConfiguration config, Container container) in c:\codelocal\chocolatey2\src\chocolatey\infrastructure.app\runners\ConsoleApplication.cs:line 91
     at chocolatey.console.Program.Main(String[] args)
  Exiting with 1
C:\projects\msbuildstructuredlog\Common.targets(51,5): error MSB3073: The command "C:\ProgramData\chocolatey\choco.exe Pack C:\projects\msbuildstructuredlog\src\StructuredLogViewer\msbuild-structured-log-viewer.nuspec -OutputDirectory C:\projects\msbuildstructuredlog\bin\Release\ -prop currentVersion=1.1.178 outdir=C:\projects\msbuildstructuredlog\bin\Release\ --debug --verbose" exited with code 1. [C:\projects\msbuildstructuredlog\src\StructuredLogViewer\StructuredLogViewer.csproj]
  Successfully created package 'C:\projects\msbuildstructuredlog\src\StructuredLogger\bin\Release\Microsoft.Build.Logging.StructuredLogger.1.1.178.nupkg'.
Command exited with code 1
```

**Comment by KirillOsenkov** [comment]
I'm reasonably confident that the file exists on disk, because the task wouldn't even run if the file didn't exist:
https://github.com/KirillOsenkov/MSBuildStructuredLog/blob/master/Common.targets#L45-L52

```
<Target Name="BuildChocolatey"
          AfterTargets="ExtractVersion"
          Condition="Exists('$(ChocolateyNuspecPath)') AND Exists('C:\ProgramData\chocolatey\choco.exe')"
          DependsOnTargets="ExtractVersion"
          Inputs="@(ChocolateyInput)"
          Outputs="$(ChocolateyNupkgPath)">
    <Exec Command="C:\ProgramData\chocolatey\choco.exe Pack $(ChocolateyNuspecPath) -OutputDirectory $(OutDir) -prop currentVersion=$(Version) outdir=$(OutDir) --debug --verbose" />
  </Target>
```

**Comment by flcdrg** [comment]
This appears to be fixed in Chocolatey 0.10.8

**Comment by ferventcoder** [comment]
Looks like this `-prop currentVersion=1.1.178 outdir=C:\projects\msbuildstructuredlog\bin\Release\`  is related to #1314, which was just released in 0.10.8. You should remove `-prop`, that is not part of the settings, just simply pass the properties with `name=value` and Chocolatey will do the right thing.
